lib/src/views/Calendar/SlideTransition.tsx

```diff
@@ -77,7 +77,7 @@ export const SlideTransition: React.SFC<SlideTransitionProps> = ({
 }) => {
   const classes = useStyles();
   if (reduceAnimations) {
-    return <div className={className}>{children}</div>;
+    return <div className={clsx(classes.root, className)}>{children}</div>;
```

Comment:
Should we spread too?

```suggestion
    return <div className={clsx(classes.root, className)} {...other}>{children}</div>;
```

Reply:
Here props are spreading to the TransitionGroup. So no, we don’t need to spread them

Reply:
Right, because SlideTransition is private so it doesn't matter